subprojects/reporting/src/main/java/org/gradle/api/reporting/Report.java

```diff
@@ -74,31 +71,6 @@ public String determineName(Report report) {
     @Input
     Property<Boolean> getRequired();
 
-    /**
-     * Whether or not this report should be generated by whatever generates it.
-     * <p>
-     * If {@code true}, the generator of this report will generate it at the appropriate time.
-     * If {@code false}, the generator of this report will not generate this report.
-     *
-     * @return Whether or not this report should be generated by whatever generates it.
-     *
-     * @deprecated Use {@link #getRequired()} instead. This method will be removed in Gradle 8.0.
-     */
-    @Deprecated
-    @ReplacedBy("required")
-    boolean isEnabled();
-
-    /**
-     * Whether or not this report should be generated by whatever generates it.
-     *
-     * @see #isEnabled()
-     * @param enabled Whether or not this report should be generated by whatever generates it.
-     *
-     * @deprecated Use {@link #getRequired()}.set() instead. This method will be removed in Gradle 8.0.
-     */
-    @Deprecated
```

Comment:
Can you also remove these from `org.gradle.api.reporting.Report.xml`?

Reply:
Do you mean to remove `required` to hide it from the docs?  `Destination` and `enabled` are already gone: https://github.com/gradle/gradle/pull/17107/files#diff-69755cd09d076c67da500a7b0b78b0e27dc7fe1d8c417c6ee85686ba36878308